src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(result, number) {
+      return result + number;
```

Comment:
The `add` method should use the arithmetic operator with an assignment operator to update the `result` property directly, rather than returning a new value.